### PR TITLE
Update crafting.lua with all legendary recipe items

### PR DESCRIPTION
**Pull request by ghost**
I updated the crafting.lua file with every available legendary recipe in the game. 
At least the ones that actually work. I named the iconic items, since that might be useful for someone?

I made small categories throughout the code, should more recipes/items be added later into the game. Figured it might be easier with some categories.

Note, this is only the legendary version of the recipes. There are still a lot more recipes in the game. But since the current command in the console is for legendary recipes (`BD.Cheats.Crafting.AddLegendaryRecipes()`), I figured I just wanted to add those. 

I hope I didn't mess anything up. Just used the in-house Github editor.

### CODE REVIEW

**Review by osulli — APPROVED** [review]
Thanks

**Comment by ghost** [comment]
I did another commit. Hopefully it's not too much of a hassle to look through :).

Edit: woops, accidentally closed it.

**Review by osulli — CHANGES_REQUESTED** [review]
Just need to update README.md with new protocols + update init.lua (if required)

**Comment by ghost** [comment]
I updated the crafting.lua once again.

Now all available/working recipes should be in there, sorted in the different quality categories.

Made a wrong pull request, now it should be included in this one lol

EDIT 1:
I don't know if the init.lua needs updating?? I'll test my version in game and see if the new commands work.

EDIT 2:
It seems all the commands work just fine in-game. All recipes are getting added correctly. At least that's what I can see.

My guess is that there's no need to update the init.lua, but I'm no expert in that regard.

File: mods/braindance_protocol/cheats/crafting.lua [hunk 1]
```diff
@@ -24,7 +24,7 @@ function Crafting.AddLegendaryMaterials(quantity)
 end
 
 function Crafting.AddLegendaryRecipes()
-    local moduleName = "Add Legendary Crafting Recipies"
+    local moduleName = "Add Legendary Crafting Recipes"
```
Comment (osulli) [inline mods/braindance_protocol/cheats/crafting.lua:27]: TIL it isn't reci-pies but reci-pes

**Review by osulli — APPROVED** [review]

**Pull request merged**